pages/api/locker/[...lockerId].ts

```diff
@@ -0,0 +1,31 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+import type { NextApiRequest, NextApiResponse } from 'next'
+import prisma from '../../../lib/prisma'
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  if (req.method === 'GET') {
+    try {
+      let { lockerId } = req.query
+      let id = lockerId?.toString() || ''
+
+      if (id == '') {
+        res.status(400).json({ message: 'Empty Request' })
+      }
+      let int_id = parseInt(id)
+
+      const lockers = await prisma.lockerBox.findMany({
+        where: {
+          lockerId: int_id
+        }
+      })
```

Comment:
this is good but i think u want to also include the locker object since it has location details. I think the query is a bit diff

```
 await prisma.locker.findUnique({
        where: {
          id: int_id
        }, 
include: {
    lockerbox: true,
location: true
  },
      })
```

something along these lines ^ 

this is bc ur querying for an actual locker here